### PR TITLE
fits: Make JVM heap size configurable

### PR DESCRIPTION
**Pull request by amayita**
This PR also improves starting & stopping services on install, upgrade
and removal of rpm packages for CentOS

Connects to https://github.com/archivematica/Issues/issues/357

### CODE REVIEW

**Review by scollazo — APPROVED** [review]
Sorry, I was too fast with the CR, can remove the start-on-boot change?

File: rpm/fits/package.spec [hunk 1]
```diff
@@ -55,15 +56,33 @@ cp log4j.properties version.properties %{buildroot}/usr/share/fits/
...
 cp -rf tools/droid  %{buildroot}/usr/share/fits/tools/
+cp -rf %{_etcdir}/sysconfig/fits  %{buildroot}/etc/sysconfig/fits
 
 
 %post
 touch /var/log/archivematica/fits.log
 chown archivematica.archivematica /var/log/archivematica/fits.log
-# TODO: reload only when the script changes. Check https://fedoraproject.org/wiki/Packaging:Scriptlets?rd=Packaging:ScriptletSnippets#Systemd
 systemctl daemon-reload
+systemctl restart fits-nailgun.service
```
Comment (scollazo) [inline rpm/fits/package.spec]: RPM packages don't usually enable their services ( [source](https://fedoraproject.org/wiki/EPEL:SysVInitScripts?rd=Packaging:SysVInitScript#Why_don.27t_we....) ). Could you remove this change?

**Review by scollazo — CHANGES_REQUESTED** [review]

**Comment by scollazo** [comment]
@amayita ping

File: rpm/fits/package.spec (hunk 1 above)
Comment (amayita) [reply rpm/fits/package.spec]: Ready!

**Comment by scollazo** [comment]
Package has been [built](https://jenkins-ci.archivematica.org/job/am-packbuild/job/rpm-jenkinsci/204/console) by jenkins and is available [here](https://jenkins-ci.archivematica.org/repos/am-packbuild/fits-heapsize/centos7/)

I'll test the package and will get back to you with my findingins.

The jenkins build is shown as failed because the makefile doesn't have the "cleanup" target, but that's out of the scope of this PR.

**Review by scollazo — APPROVED** [review]
Better late than never, let get this merged!

**Review by mamedin — APPROVED** [review]

**Comment by scollazo** [comment]
Now, the repository for 1.11.x needs to be created, and the package needs to be built and upload to https://packages.archivematica.org/1.11.x/ubuntu-externals  

If you use https://jenkins-ci.archivematica.org/job/am-packbuild/ to build the package, please copy the link to the log build here.

**Pull request merged**